docker-compose.yml

```diff
@@ -29,17 +49,29 @@ services:
       MODE: 'dev'
     networks:
       - tbapi
+
+  nginx:
```

Comment:
Don't think it's entirely necessary to serve static files separately (gunicorn/flask do a good enough job), unless it would be more work to remove